game.py

```diff
@@ -19,7 +19,7 @@ def play_game(self):
         print('Player 2 what is your name')
         self.players.append(Player('X'))
         while True:
-             try:
+             try: # while loop
```

Comment:
Your try/except block should be just around the methods that you know will raise exceptions (add_piece and get_choice).  If another part of your program has a bug, you want it to crash so you can fix the bug.